stable/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM frolvlad/alpine-glibc:alpine-3.12
```

Comment:
I don't want to have yet another copy of the same files. It would be better if we told the (over complicated) CI script to properly handle this.

Reply:
OK, I see. I'm not very comfortable with writing (readable) bash code though, and changing the CI code would make the PR more heavy (I wouldn't call it heavy right now, as it most of it is C&P).

Are you comfortable with having Python code as CI script?
I could happily refactor that CI script to Python, but I don't want to put much effort into a PR that is getting rejected afterwards

Reply:
> Are you comfortable with having Python code as CI script?

The CI script needs a rewrite but Factorio won't get to many updates in the near future so I was avoiding it.

> I could happily refactor that CI script to Python, but I don't want to put much effort into a PR that is getting rejected afterwards

I am not sure how to properly do that. If you have an idea let me know.